src/index.css

```diff
@@ -11,3 +11,37 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.header {
```

Comment:
I would recommend using the global variables contained in the styles/variables/border.scss file in addition to the global brand color variables used in styles/variables/colors.scss. This should this also be saved as a .scss file and imported in to the appropriate section of dotgov.scss.

Reply:
So the body and code css isnt something that I created (I think these are react defaults) and Im actually not sure what they are and will most likely be ignored/removed if this project starts using SASS. As far as incorporating these into dotgov.scss I think that is a good idea but Marty wasn't sure how he wanted to handle the css for this project. Either importing from dotgov-components or having its own separate css files. Depending on that decision then we can move accordingly.